hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/graphite/GraphiteClient.java

```diff
@@ -32,11 +32,10 @@ public GraphiteClient(WebTarget webTarget) {
     public GraphiteMetrics readMetrics(String... metrics) {
         try {
             GraphiteMetrics response = new GraphiteMetrics();
-            queryGraphite(metrics).stream()
-                    .forEach(metric -> response.addMetricValue(metric.getTarget(), getFirstValue(metric)));
+            queryGraphite(metrics).stream().forEach(metric -> response.addMetricValue(metric.getTarget(), getFirstValue(metric)));
             return response;
         } catch (GraphiteConnectionException exception) {
-            logger.warn("Unable to read from Graphite", exception);
+            logger.warn("Unable to read from Graphite", exception.getMessage());
```

Comment:
I decided to remove whole stack trace because it's spamming our logs lately ;) But you have right that last exception cause message doesn't say much, so I decided to log root cause message which always tells you something informative. Right now we will be see something like this:
```
p.a.t.h.m.i.graphite.GraphiteClient - Unable to read from Graphite. Connection refused
```